tests/cli/workflow/mip/conftest.py

```diff
@@ -15,6 +27,209 @@ def mock_root_folder():
     return "/var/empty"
 
 
+@pytest.fixture(name="mip_case_ids")
+def mip_case_ids() -> dict:
+    """Dictionary of case ids, connected samples, their name and if they should fail (textbook or not)"""
+
+    case_ids = {
+        "yellowhog": {
+            "textbook": True,
+            "name": "F0000001",
+            "internal_id": "ACC00000",
+        },
+        "purplesnail": {
+            "textbook": False,
+            "name": "F0000003",
+            "internal_id": "ACC00000",
+        },
+        "bluezebra": {
+            "textbook": True,
+            "name": "F0000002",
+            "internal_id": "ACC00000",
+        },
+    }
+
+    return case_ids
+
+
+@pytest.fixture(name="mip_case_dirs")
+def fixture_mip_case_dirs(mip_case_ids: dict, project_dir: Path) -> dict:
+    """Create case directories and return a dictionary of the tmpdir paths"""
+
+    mip_case_dirs = {}
+
+    for case in mip_case_ids:
+        case_dir = Path(project_dir / "cases" / case)
+        case_dir.mkdir(exist_ok=True, parents=True)
+        mip_case_dirs[case] = case_dir
+
+    return mip_case_dirs
+
+
+@pytest.fixture(name="mip_deliverables")
+def make_mip_deliverables(
```

Comment:
```suggestion
def fixture_make_mip_deliverables(
```
Although base on the name it sounds like this should rather be a function than a fixture, right?

Reply:
I think its better to call it fixture_mip_deliverables, I will change it for the other ones as well in another commit